inc/Engine/Optimization/Minify/JS/AbstractJSOptimization.php

```diff
@@ -146,23 +146,21 @@ protected function get_minify_url( $filename, $original_url = '' ) {
 	 * @since  3.1
 	 * @author Remy Perona
 	 *
-	 * @return bool|string
+	 * @return array
 	 */
-	protected function get_jquery_url() {
+	protected function get_jquery_urls() {
 		global $wp_scripts;
 
 		if ( ! $this->options->get( 'defer_all_js', 0 ) || ! $this->options->get( 'defer_all_js_safe', 0 ) ) {
-			return false;
-		}
-
-		if ( ! isset( $wp_scripts->registered['jquery-core']->src ) ) {
-			return false;
+			return [];
 		}
-
-		if ( '' === wp_parse_url( $wp_scripts->registered['jquery-core']->src, PHP_URL_HOST ) ) {
-			return wp_parse_url( site_url( $wp_scripts->registered['jquery-core']->src, PHP_URL_PATH ) );
-		}
-
-		return $wp_scripts->registered['jquery-core']->src;
+		$jquery           = site_url( wp_scripts()->registered['jquery-core']->src );
```

Comment:
- We already have the global `$wp_scripts` at the start of the method, is there a reason to use `wp_scripts()` instead?
- Did you test that the returned value from the `src` property is always a path, and not an absolute URL? Else the URL could be incorrect

Reply:
removed the global and revisited the code. It's an url and I am using `rocket_clean_exclude_file()`  function to ged rid of the domain and grab only the path.